### DISCUSSION TITLE
Pedal Animation (Brakes)

### DISCUSSION THREAD

**Discussion by Viper787** [magknight/787aviatorsTracker#571] [Completed things] [answered]
**Describe the bug**
When operating foot pedals rudder animation works, but braking animation does not.

**To Reproduce**
Use pedals and watch the animation in cockpit, pressing the brakes do not show the forward/backward pivot motion of the pedal. 

**Expected behavior**
Brake pedals should show animation

**Screenshots**

**Desktop (please complete the following information):**
 - OS: mac 10.12.6
 - X-Plane version 11.50r3
 - Version 1.68 beta2

**Additional context**

**Comment by aeroniemi** [comment, marked as answer]
Released as part of this version
